sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/FluxTraceV2.java

```diff
@@ -0,0 +1,71 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
In general, I was wondering if we _need_ all the new v2 classes. Is it possible to choose to use the 2nd version in constructor or is there a lot of different logic? I ask because it'll be easier to trace history if we don't delete a class entirely to replace it.

Reply:
We need most of these new types due to logic/usage/design change. When possible, we used Ctr overloads or added new methods to existing types.

The ReceiveLinkHandler2.java, FluxTraceV2 are only two types that are kind of duplicate types with some extra code, plan is to keep the history by using original names when we remove v1 switch. The AutoDispositionLockRenew is the remaining type but is marked as deprecated due to known problems with Auto* in low level clients.